app/src/main/java/com/example/dine_and_donate/Models/Notifications.java

```diff
@@ -0,0 +1,43 @@
+package com.example.dine_and_donate.Models;
+
+public class Notifications {
```

Comment:
this is just a single Notification right? I think the name should be singular then, not plural